manly.py

```diff
@@ -155,4 +149,12 @@ def main(command):
     if not len(args.command):
         print("manly: missing COMMAND\n" "Try 'manly --help' for more information.")
     else:
-        main(args.command)
+        title, output=main(args.command)
```

Comment:
If its not too much trouble, run `black` to format the code before commiting to avoid these formatting errors (missing space around `=`)

https://github.com/ambv/black

Reply:
Added space. (Formatted using `black`).